bika/lims/content/attachment.py

```diff
@@ -133,35 +138,38 @@ def Title(self):
     def getTextTitle(self):
         """Return the request and possibly analayis title as title
         """
-        requestid = self.getRequestID()
-        if requestid:
-            analysis = self.getAnalysis()
-            if analysis:
-                return '%s - %s' % (requestid, analysis.Title())
-            else:
-                return requestid
-        else:
+        request_id = self.getRequestID()
+        if not request_id:
             return None
```

Comment:
Better return an empty string here, so it fit's better with the return type in line 149